tests/integration/test_integration.py

```diff
@@ -290,6 +290,7 @@ def test_metrics(encoding, monkeypatch):
 
 
 @allencodings
+@pytest.mark.skipif(AGENT_VERSION == "testagent", reason="FIXME: Test agent doesn't support this for some reason.")
```

Comment:
If this is a newly introduced incompatibility, I'd recommend opening a ticket on the test agent to track it.